dvc/commands/plots.py

```diff
@@ -152,16 +159,16 @@ class CmdPlotsTemplates(CmdBase):
 
     def run(self):
         import os
-
+        from dvc_render.vega_templates import dump_templates
```

Comment:
I considered that the logic for dumping and loading templates made more sense in `dvc-render`. The only link between `repo` and the templates is now `repo.plots.templates_dir`